### DISCUSSION TITLE
Inquiry about RoCC (Rocket Chip Coprocessor) Support

### DISCUSSION THREAD

**Discussion by GGKOP** [gem5/gem5#467] [Q&A]
I am currently working on adapting boom to gem5 and exploring the possibility of adding an NPU (Neural Processing Unit) module. In this context, I wanted to inquire about the support for RoCC (Rocket Chip Coprocessor) in gem5.

Is gem5 equipped to handle RoCC, which is a co-processor of the Rocket-core and can be controlled through custom RISC-V extension instructions?

**Comment by BobbyRBruce** [comment]
I moved this from or Issues as it's not a big report or feature suggestion. The gem5 Discussion's board is a better place to ask questions.

**Comment by momimmu** [comment]
Hello, have you solved this issue? I'm struggling with the same one.
